src/cli.js

```diff
@@ -11,8 +10,9 @@ program
   .parse(process.argv);
 
 const bundle = JSON.parse(fs.readFileSync(program.bundle, 'utf8'));
-const mainELM = loadELM(bundle);
-logger.info(`exporting file with name: ${mainELM.mainLibrary.library.identifier.id}.json`);
-const moduleJSON = exportModule(mainELM);
-fs.writeFileSync(`${mainELM.mainLibrary.library.identifier.id}.json`, JSON.stringify(moduleJSON));
-logger.info(`${mainELM.mainLibrary.library.identifier.id}.json was exported successfully`);
+const generator = new Generator(bundle, true);
```

Comment:
```suggestion
const generator = new Generator(bundle, false);
```

This is my fault. Pretty sure I told you the wrong boolean